manifest.json

```diff
@@ -2994,6 +2994,55 @@
                 }
             }
         },
+        "Banane9.StopDisappearingLocomotionMenu": {
```

Comment:
contains funky IL stuff, will need to review properly later.

Reply:
Fukit it's probably fine, the IL includes some funny jump/call instructions but don't see how they would do anything malicious
![seal](https://user-images.githubusercontent.com/5854483/184557817-331b1bc4-6ee8-428e-b399-aca7a770767a.jpg)
.

Reply:
It's just some extra if stuff in IL 👀